docs/modify/zodan/zodan_tutorial.md

```
@@ -74,19 +78,28 @@ psql -d inventory -c "CREATE EXTENSION spock;"
-**Using the Zodan Procedure to Add a Node** 
+## Using the Zodan Procedure to Add a Node
+
+After creating the node, you can use [Zodan scripts](zodan_readme.md) to
+simplify adding a node to a cluster.
+
+To use the SQL script, connect to the new node that you wish to add to the
+pgEdge cluster. In the following example, the `psql` command connects to the
+new node:
 
-After creating the node, you can use [Zodan scripts](zodan_readme.md) to simplify adding a node to a cluster. To use the SQL script, connect to the new node that you wish to add to the pgedge cluster:
 ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)